test/test.js

```diff
@@ -19,34 +19,41 @@ var OAuth2 = gas.require('./src', mocks, options);
 
 describe('Service', function() {
   describe('#getToken()', function() {
-    it('should load from the in-memory store', function() {
+    it('should return null when no token is stored', function() {
+      var service = OAuth2.createService('test')
+        .setPropertyStore(new MockProperties())
```

Comment:
Note: Normally 2x (=4 spaces) the normal indentation is used for statement continuation.

Reply:
Done, although es-lint apparently can't catch this.